src/PostgREST/AppState.hs

```diff
@@ -97,7 +97,7 @@ destroy = destroyPool
 
 initPool :: AppConfig -> IO SQL.Pool
 initPool AppConfig{..} =
-  SQL.acquire configDbPoolSize Nothing $ toUtf8 configDbUri
+  SQL.acquire configDbPoolSize ((1000000 *) <$> configDbPoolAcquisitionTimeout) $ toUtf8 configDbUri
```

Comment:
Just for clarity
```suggestion
  let microSecond = 1000000 in
  SQL.acquire configDbPoolSize ((microSecond *) <$> configDbPoolAcquisitionTimeout) $ toUtf8 configDbUri
```

Reply:
I've addressed this (with slightly different naming, `microSecond = 1000000` didn't feel quite right because that's not literally a microsecond, but rather a second in microseconds...).